dev/com.ibm.ws.jsp/src/com/ibm/wsspi/jsp/taglib/config/TldPathConfig.java

```diff
@@ -74,4 +74,46 @@ public void setUri(String string) {
     public boolean containsListenerDefs() {
         return containsListenerDefs;
     }
+
+    public boolean isCustomURIUsed() {
+        return this.forceCustomURI;
+    }
+
+    /*
+     * Ignore the URI set in the TLD attribute and used the URI 
+     * specified via the TldPathConfig constructor
+     * @return - currrent TldPathConfig instance
+     */
+    public TldPathConfig forceCustomURI() {
+        this.forceCustomURI = true;
+        return this;
+    }
+
+    /*
+     * Ignore the URI set in the TLD attribute and used the URI 
```

Comment:
`and use the URI`

Reply:
Also the javadoc for this and `forceCustomURI` seem to say basically the same thing. I'm a bit confused reading it.